exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
+      staging: ${drill.exec.zk.root}"/udf/staging",
```

Comment:
Regarding the dfs/local file system... A useful model would be that files are stored EITHER locally OR in DFS. Local works only for NFS-mounted directories (or single-node test/demo configurations.) In this model, we still use the Hadoop FileSystem class, but we use either file://foo or hdfs://foo. (I'm a bit hazy on the details of how this works.)

We'll discuss this issue elsewhere as it goes beyond a code review comment.